scripts/src/models/orders.ts

```diff
@@ -59,9 +59,9 @@ function updateQueryWithFilter(query: SelectQueryBuilder<any>, name: string, val
 	const fieldName = alias ? `${ alias }.${ name }` : name;
 
 	if (value.startsWith("!")) {
-		query.andWhere(`${ fieldName } != :value`, { value: value.substring(1) });
+		query.andWhere(`${ fieldName } != :${ fieldName }`, { [fieldName]: value.substring(1) });
```

Comment:
i find this to be pretty confusing, maybe:
```
const fieldValue = `${ fieldName }_value`;
query.andWhere(`${ fieldName } != :${ fieldValue }`, { [fieldValue]: value.substring(1) });
```

Reply:
I don't necessarily agree.

Reply:
@doodyparizada 
on what part?